issues.txt

```diff
@@ -0,0 +1 @@
+1)When we clear first timer the second timer should also be cleareed right
```

Comment:
Remove this file, you can put your doubts in the comments of the PR. Or maybe in the communication channel.